src/nemo_evaluator/adapters/interceptors/endpoint.py

```
@@ -128,6 +131,62 @@ def _request_stream_usage_chunks(path: str, body: dict[str, Any]) -> None:
         elif isinstance(stream_options, dict) and "include_usage" not in stream_options:
             body["stream_options"] = {**stream_options, "include_usage": True}
 
+    @staticmethod
+    def _is_streaming_chat_request(path: str, body: dict[str, Any]) -> bool:
+        return body.get("stream") is True and path.rstrip("/").endswith("/chat/completions")
+
+    @staticmethod
+    async def _post_json(
+        session: aiohttp.ClientSession, url: str, headers: dict[str, str], body: dict[str, Any]
+    ) -> Any:
```

Ruff (0.15.21)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_post_json`

(ANN401)